templates/web/pages/org_form/org_form.tt.html

```diff
@@ -32,6 +33,10 @@ <h2>[% lang("organization_members") %]</h2>
 			[% FOREACH users IN org_members %]
 			<tr>
 				<td>[% count %].</td>
+				<td>
+					<input type="checkbox" name="admin_status_[% users.userid %]" value= "1" [% IF admin_status.users.userid %]checked[% END %]>
```

Comment:
idk why but the condition inside if block( "admin_status.users.userid")  is not working i.e the checkbox is not already checked for admins.